synapse/storage/databases/main/schema/delta/60/01refresh_tokens.sql.sqlite

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE refresh_tokens (
```

Comment:
Don't we also need a `postgres` schema change?

Reply:
That's done now.